pad/editor/src/lib.rs

```diff
@@ -744,6 +757,8 @@ pub fn Editor<'a>(
             "z" if os_ctrl(event) => state.update(|state| state.undo()),
             // Insert # Experimental! comment
             "e" if os_ctrl(event) => insert_experimental(),
+            // Remove # Experimental! comment
+            "E" if os_ctrl(event) => remove_experimental(),
```

Comment:
```suggestion
            // Remove # Experimental! comment
            "e" | "E" if os_ctrl(event) && event.shift_key() => remove_experimental(),
            // Insert # Experimental! comment
            "e" if os_ctrl(event) => insert_experimental(),
```
At least on my machine, the ⇧⌘E input was not triggering the functionality without this change. The new match branch must be above the old one, since the latter catches more cases. See the undo/redo bindings above.